mne/annotations.py

```diff
@@ -1140,7 +1141,13 @@ def _write_annotations_txt(fname, annot):
 
 @fill_doc
 def read_annotations(
-    fname, sfreq="auto", uint16_codec=None, encoding="utf8", ignore_marker_types=False
+    fname,
+    sfreq="auto",
```

Comment:
We've started inserting these in reasonable places. This seems like a good choice here
```suggestion
    sfreq="auto",
    *,
```